src/zscale.jl

```diff
@@ -0,0 +1,248 @@
+MAX_REJECT = 0.5
+MIN_NPIXELS = 5
+GOOD_PIXEL = 0
+BAD_PIXEL = 1
+KREJ = 2.5
+MAX_ITERATIONS = 5
+
+
+function flatten(image)
+
+    # colapses into one dimension (Float64) a 2D array in row-major (C-style flattening)
+
+    rows = size(image)[1]
+    cols = size(image)[2]
+    n_pxls = rows * cols
+
+    result = Array{Float64}(undef, n_pxls)
+    
+    k = 1
+    for i in 1:rows, j in 1:cols
+        result[k] = image[i, j]
+        k += 1
+    end
+
+    return result
+end
+
+
+function zsc_sample(image, maxpix)
```

Comment:
What is this function actually doing and how?  Please add docstrings to functions.